### PR TITLE
Fix append syntax in datastreamsgo README

### PR DESCRIPTION
**Pull request by DanielJamieson**
I noticed that the syntax in the example is incorrect. Here is the fix

### CODE REVIEW

**Review by kr-igor — APPROVED** [review]

**Comment by kr-igor** [comment]
Thanks for fixing this!

**Comment by DanielJamieson** [comment]
> Thanks for fixing this!

It's a pleasure :) I don't have write access so I can't merge the PR, so I leave it for you or another maintainer @kr-igor

**Pull request merged**
